src/mmw/js/src/analyze/controllers.js

```diff
@@ -20,7 +20,29 @@ var AnalyzeController = {
             App.map.set('maskLayerApplied', true);
         }
 
-        if (!settings.get('activityMode')) {
+        if (settings.get('activityMode')) {
+            // Only one project allowed in Activity Mode. Save current project
+            // and if in embedded mode, update interactive state for container.
+            var project = App.currProject,
+                map = App.map;
```

Comment:
Yes. `project` is referred to a number of times, but `map` only twice. The second is a bit of a relic since this code was originally came from a place where `map` was a function parameter, now we source it from `App`.